packages/blockchain-utils-linking/src/util.ts

```diff
@@ -22,6 +22,14 @@ export interface ConsentMessage {
   timestamp?: number
 }
 
+export interface CapabilityOpts {
+  domain?: string
+  statement?: string
+  version?: string
+  nonce?: string
+  requestId?: string
+}
```

Comment:
```suggestion
  requestId?: string
  expirationTime? string
  resources? Array<string>
}
```

Reply:
don't want resources directly as a function argument?

Reply:
Well people might want to include resources that are not streamIds :)

Reply:
hmmmmmm